common/util/BUILD

```diff
@@ -658,3 +658,22 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_library(
+    name = "sha256",
+    srcs = ["sha256.cc"],
+    hdrs = ["sha256.h"],
+    deps = [
+        "@com_google_absl//absl/strings",
+    ],
```

Comment:
The code looks like some sort of MIT license. Do we need to annotate this library with some different applicable license here in the cc_library ? (the [`default_applicable_license`](https://github.com/chipsalliance/verible/blob/0b8cb4e0b1e95b2431239a859dd5f4a29d346b32/common/util/BUILD#L5) on the package-level points to [apache](https://github.com/chipsalliance/verible/blob/0b8cb4e0b1e95b2431239a859dd5f4a29d346b32/BUILD#L16-L21))

(but I am also fine with deferring this decision a little bit until we talked to the licensing experts and fix it then)

Reply:
I wasn't sure what to do here. Forwarding the original license seemed like a reasonable thing to do.
I can check with the licensing experts.